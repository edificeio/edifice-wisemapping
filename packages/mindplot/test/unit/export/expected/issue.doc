<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<?mso-application progid="Word.Document"?><w:wordDocument xmlns:sl="http://schemas.microsoft.com/schemaLibrary/2003/core" xmlns:w="http://schemas.microsoft.com/office/word/2003/wordml" xmlns:w10="urn:schemas-microsoft-com:office:word" xmlns:v="urn:schemas-microsoft-com:vml" xmlns:dt="uuid:C2F41010-65B3-11d1-A29F-00AA00C14882" xmlns:wx="http://schemas.microsoft.com/office/word/2003/auxHint" xmlns:aml="http://schemas.microsoft.com/aml/2001/core" xmlns:o="urn:schemas-microsoft-com:office:office">
<o:DocumentProperties>
<o:Title>La computadora</o:Title>
</o:DocumentProperties>
<w:styles>
<w:versionOfBuiltInStylenames w:val="4"/>
<w:latentStyles w:defLockedState="off" w:latentStyleCount="156"/>
<w:style w:type="paragraph" w:default="on" w:styleId="Normal">
<w:name w:val="Normal"/>
<w:rsid w:val="00831C9D"/>
<w:rPr>
<wx:font wx:val="Times New Roman"/>
<w:sz w:val="24"/>
<w:sz-cs w:val="24"/>
<w:lang w:val="EN-US" w:fareast="EN-US" w:bidi="AR-SA"/>
</w:rPr>
</w:style>
<w:style w:type="paragraph" w:styleId="Heading1">
<w:name w:val="heading 1"/>
<wx:uiName wx:val="Heading 1"/>
<w:basedOn w:val="Normal"/>
<w:next w:val="Normal"/>
<w:rsid w:val="00BA7540"/>
<w:pPr>
<w:pStyle w:val="Heading1"/>
<w:keepNext/>
<w:spacing w:before="240" w:after="60"/>
<w:outlineLvl w:val="0"/>
</w:pPr>
<w:rPr>
<w:rFonts w:ascii="Arial" w:h-ansi="Arial" w:cs="Arial"/>
<wx:font wx:val="Arial"/>
<w:b/>
<w:b-cs/>
<w:kern w:val="32"/>
<w:sz w:val="32"/>
<w:sz-cs w:val="32"/>
</w:rPr>
</w:style>
<w:style w:type="paragraph" w:styleId="Heading2">
<w:name w:val="heading 2"/>
<wx:uiName wx:val="Heading 2"/>
<w:basedOn w:val="Normal"/>
<w:next w:val="Normal"/>
<w:rsid w:val="00BA7540"/>
<w:pPr>
<w:pStyle w:val="Heading2"/>
<w:keepNext/>
<w:spacing w:before="240" w:after="60"/>
<w:outlineLvl w:val="1"/>
</w:pPr>
<w:rPr>
<w:rFonts w:ascii="Arial" w:h-ansi="Arial" w:cs="Arial"/>
<wx:font wx:val="Arial"/>
<w:b/>
<w:b-cs/>
<w:i/>
<w:i-cs/>
<w:sz w:val="28"/>
<w:sz-cs w:val="28"/>
</w:rPr>
</w:style>
<w:style w:type="paragraph" w:styleId="Heading3">
<w:name w:val="heading 3"/>
<wx:uiName wx:val="Heading 3"/>
<w:basedOn w:val="Normal"/>
<w:next w:val="Normal"/>
<w:rsid w:val="00BA7540"/>
<w:pPr>
<w:pStyle w:val="Heading3"/>
<w:keepNext/>
<w:spacing w:before="240" w:after="60"/>
<w:outlineLvl w:val="2"/>
</w:pPr>
<w:rPr>
<w:rFonts w:ascii="Arial" w:h-ansi="Arial" w:cs="Arial"/>
<wx:font wx:val="Arial"/>
<w:b/>
<w:b-cs/>
<w:sz w:val="26"/>
<w:sz-cs w:val="26"/>
</w:rPr>
</w:style>
<w:style w:type="paragraph" w:styleId="Heading4">
<w:name w:val="heading 4"/>
<wx:uiName wx:val="Heading 4"/>
<w:basedOn w:val="Normal"/>
<w:next w:val="Normal"/>
<w:rsid w:val="00BA7540"/>
<w:pPr>
<w:pStyle w:val="Heading4"/>
<w:keepNext/>
<w:spacing w:before="240" w:after="60"/>
<w:outlineLvl w:val="3"/>
</w:pPr>
<w:rPr>
<wx:font wx:val="Times New Roman"/>
<w:b/>
<w:b-cs/>
<w:sz w:val="28"/>
<w:sz-cs w:val="28"/>
</w:rPr>
</w:style>
<w:style w:type="paragraph" w:styleId="Heading5">
<w:name w:val="heading 5"/>
<wx:uiName wx:val="Heading 5"/>
<w:basedOn w:val="Normal"/>
<w:next w:val="Normal"/>
<w:rsid w:val="00BA7540"/>
<w:pPr>
<w:pStyle w:val="Heading5"/>
<w:spacing w:before="240" w:after="60"/>
<w:outlineLvl w:val="4"/>
</w:pPr>
<w:rPr>
<wx:font wx:val="Times New Roman"/>
<w:b/>
<w:b-cs/>
<w:i/>
<w:i-cs/>
<w:sz w:val="26"/>
<w:sz-cs w:val="26"/>
</w:rPr>
</w:style>
<w:style w:type="paragraph" w:styleId="Heading6">
<w:name w:val="heading 6"/>
<wx:uiName wx:val="Heading 6"/>
<w:basedOn w:val="Normal"/>
<w:next w:val="Normal"/>
<w:rsid w:val="00BA7540"/>
<w:pPr>
<w:pStyle w:val="Heading6"/>
<w:spacing w:before="240" w:after="60"/>
<w:outlineLvl w:val="5"/>
</w:pPr>
<w:rPr>
<wx:font wx:val="Times New Roman"/>
<w:b/>
<w:b-cs/>
<w:sz w:val="22"/>
<w:sz-cs w:val="22"/>
</w:rPr>
</w:style>
<w:style w:type="paragraph" w:styleId="Heading7">
<w:name w:val="heading 7"/>
<wx:uiName wx:val="Heading 7"/>
<w:basedOn w:val="Normal"/>
<w:next w:val="Normal"/>
<w:rsid w:val="00BA7540"/>
<w:pPr>
<w:pStyle w:val="Heading7"/>
<w:spacing w:before="240" w:after="60"/>
<w:outlineLvl w:val="6"/>
</w:pPr>
<w:rPr>
<wx:font wx:val="Times New Roman"/>
</w:rPr>
</w:style>
<w:style w:type="paragraph" w:styleId="Heading8">
<w:name w:val="heading 8"/>
<wx:uiName wx:val="Heading 8"/>
<w:basedOn w:val="Normal"/>
<w:next w:val="Normal"/>
<w:rsid w:val="00BA7540"/>
<w:pPr>
<w:pStyle w:val="Heading8"/>
<w:spacing w:before="240" w:after="60"/>
<w:outlineLvl w:val="7"/>
</w:pPr>
<w:rPr>
<wx:font wx:val="Times New Roman"/>
<w:i/>
<w:i-cs/>
</w:rPr>
</w:style>
<w:style w:type="paragraph" w:styleId="Heading9">
<w:name w:val="heading 9"/>
<wx:uiName wx:val="Heading 9"/>
<w:basedOn w:val="Normal"/>
<w:next w:val="Normal"/>
<w:rsid w:val="00BA7540"/>
<w:pPr>
<w:pStyle w:val="Heading9"/>
<w:spacing w:before="240" w:after="60"/>
<w:outlineLvl w:val="8"/>
</w:pPr>
<w:rPr>
<w:rFonts w:ascii="Arial" w:h-ansi="Arial" w:cs="Arial"/>
<wx:font wx:val="Arial"/>
<w:sz w:val="22"/>
<w:sz-cs w:val="22"/>
</w:rPr>
</w:style>
<w:style w:type="character" w:default="on" w:styleId="DefaultParagraphFont">
<w:name w:val="Default Paragraph Font"/>
<w:semiHidden/>
</w:style>
<w:style w:type="table" w:default="on" w:styleId="TableNormal">
<w:name w:val="Normal Table"/>
<wx:uiName wx:val="Table Normal"/>
<w:semiHidden/>
<w:rPr>
<wx:font wx:val="Times New Roman"/>
</w:rPr>
<w:tblPr>
<w:tblInd w:w="0" w:type="dxa"/>
<w:tblCellMar>
<w:top w:w="0" w:type="dxa"/>
<w:left w:w="108" w:type="dxa"/>
<w:bottom w:w="0" w:type="dxa"/>
<w:right w:w="108" w:type="dxa"/>
</w:tblCellMar>
</w:tblPr>
</w:style>
<w:style w:type="list" w:default="on" w:styleId="NoList">
<w:name w:val="No List"/>
<w:semiHidden/>
</w:style>
<w:style w:type="paragraph" w:styleId="Title">
<w:name w:val="Title"/>
<w:basedOn w:val="Normal"/>
<w:rsid w:val="00BA7540"/>
<w:pPr>
<w:pStyle w:val="Title"/>
<w:spacing w:before="240" w:after="60"/>
<w:jc w:val="center"/>
<w:outlineLvl w:val="0"/>
</w:pPr>
<w:rPr>
<w:rFonts w:ascii="Arial" w:h-ansi="Arial" w:cs="Arial"/>
<wx:font wx:val="Arial"/>
<w:b/>
<w:b-cs/>
<w:kern w:val="28"/>
<w:sz w:val="32"/>
<w:sz-cs w:val="32"/>
</w:rPr>
</w:style>
<w:style w:type="paragraph" w:styleId="BodyText">
<w:name w:val="Body Text"/>
<w:basedOn w:val="Normal"/>
<w:rsid w:val="00BA7540"/>
<w:pPr>
<w:pStyle w:val="BodyText"/>
<w:spacing w:after="120"/>
</w:pPr>
<w:rPr>
<wx:font wx:val="Times New Roman"/>
</w:rPr>
</w:style>
</w:styles>
<w:body>
<wx:sect>
    <wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Title"/>
</w:pPr>
<w:r>
<w:t>La computadora</w:t>
</w:r>
</w:p>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading1"/>
</w:pPr>
<w:r>
<w:t>Software (Programas y datos con los que funciona la computadora)</w:t>
</w:r>
</w:p>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading2"/>
</w:pPr>
<w:r>
<w:t>Software de Sistema:Permite el entendimiento entre el usuario y la maquina.</w:t>
</w:r>
</w:p>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>Microsoft Windows</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>GNU/LINUX</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>MAC</w:t>
</w:r>
</w:p>
</wx:sub-section>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading2"/>
</w:pPr>
<w:r>
<w:t>Software de Aplicación: Permite hacer hojas de calculo navegar en internet, base de datos, etc.</w:t>
</w:r>
</w:p>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>Office</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>Libre Office</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>Navegadores</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>Msn</w:t>
</w:r>
</w:p>
</wx:sub-section>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading2"/>
</w:pPr>
<w:r>
<w:t>Software de Desarrollo</w:t>
</w:r>
</w:p>
</wx:sub-section>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading1"/>
</w:pPr>
<w:r>
<w:t>Hardware (componentes físicos)</w:t>
</w:r>
</w:p>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading2"/>
</w:pPr>
<w:r>
<w:t>Entrada de datos</w:t>
</w:r>
</w:p>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>Ratón, Teclado, Joystick, Cámara digital, Micrófono, Escáner.</w:t>
</w:r>
</w:p>
</wx:sub-section>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading2"/>
</w:pPr>
<w:r>
<w:t>Salida de datos</w:t>
</w:r>
</w:p>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>Monitor, Impresora, Bocinas, Plóter.</w:t>
</w:r>
</w:p>
</wx:sub-section>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading2"/>
</w:pPr>
<w:r>
<w:t>Almacenamiento</w:t>
</w:r>
</w:p>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>Disquete, Disco compacto, DVD, BD, Disco duro, Memoria flash.</w:t>
</w:r>
</w:p>
</wx:sub-section>
</wx:sub-section>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading1"/>
</w:pPr>
<w:r>
<w:t>Tipos de computadora</w:t>
</w:r>
</w:p>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading2"/>
</w:pPr>
<w:r>
<w:t>Computadora personal de escritorio o Desktop</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading2"/>
</w:pPr>
<w:r>
<w:t>PDA</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading2"/>
</w:pPr>
<w:r>
<w:t>Laptop</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading2"/>
</w:pPr>
<w:r>
<w:t>Servidor</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading2"/>
</w:pPr>
<w:r>
<w:t>Tablet PC</w:t>
</w:r>
</w:p>
</wx:sub-section>
</wx:sub-section>
</wx:sub-section>
</wx:sect>
</w:body>
</w:wordDocument>
