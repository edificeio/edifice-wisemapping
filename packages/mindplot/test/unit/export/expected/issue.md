# La computadora

- Hardware
(componentes físicos)
	- Entrada de datos

		-  Ratón, Teclado, Joystick,
Cámara digital, Micrófono, Escáner.
	- Salida de datos
		- Monitor, Impresora, Bocinas, Plóter.

	- Almacenamiento
		- Disquete, Disco compacto, DVD,
BD, Disco duro, Memoria flash.
- Software
(Programas y datos con los que funciona la computadora)

	- Software de Sistema:Permite el entendimiento
entre el usuario y la maquina.
		- Microsoft Windows
		- GNU/LINUX
		- MAC 
	- Software de Aplicación: Permite hacer hojas de
calculo navegar en internet, base de datos, etc.
		- Office
		- Libre Office
		- Navegadores
		- Msn
	- Software de Desarrollo

- Tipos de computadora
	- Computadora personal de escritorio o Desktop
	- PDA

	- Laptop
	- Servidor
	- Tablet PC

