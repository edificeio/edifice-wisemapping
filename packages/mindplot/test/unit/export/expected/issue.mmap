<?xml version="1.0" encoding="UTF-8"?><ap:Map xmlns:ap="http://schemas.mindjet.com/MindManager/Application/2003"><ap:OneTopic><ap:Topic OId="1W54nezC90m8NYAi2fjQvw=="><ap:SubTopics><ap:Topic OId="5954nezC90m8NYAi2fjQvw=="><ap:SubTopics><ap:Topic OId="9254nezC90m8NYAi2fjQvw=="><ap:SubTopics><ap:Topic OId="1014nezC90m8NYAi2fjQvw=="><ap:Text PlainText="Microsoft Windows"><ap:Font Color="ff000000" Size="12"/></ap:Text><ap:Color FillColor="ffffd966"/><ap:SubTopicShape/></ap:Topic><ap:Topic OId="1064nezC90m8NYAi2fjQvw=="><ap:Text PlainText="GNU/LINUX"><ap:Font Color="ff000000" Size="12"/></ap:Text><ap:Color FillColor="ffffd966"/><ap:SubTopicShape/></ap:Topic><ap:Topic OId="1074nezC90m8NYAi2fjQvw=="><ap:Text PlainText="MAC "><ap:Font Color="ff000000" Size="12"/></ap:Text><ap:Color FillColor="ffffd966"/><ap:SubTopicShape/></ap:Topic></ap:SubTopics><ap:Text PlainText=""><ap:Font Color="ff000000" Bold="true" Size="12"/></ap:Text><ap:Color FillColor="fff1c232"/><ap:SubTopicShape/></ap:Topic><ap:Topic OId="9354nezC90m8NYAi2fjQvw=="><ap:SubTopics><ap:Topic OId="1084nezC90m8NYAi2fjQvw=="><ap:Text PlainText="Office"><ap:Font Color="ff000000" Size="12"/></ap:Text><ap:Color FillColor="ffffd966"/><ap:SubTopicShape/></ap:Topic><ap:Topic OId="1094nezC90m8NYAi2fjQvw=="><ap:Text PlainText="Libre Office"><ap:Font Color="ff000000" Size="12"/></ap:Text><ap:Color FillColor="ffffd966"/><ap:SubTopicShape/></ap:Topic><ap:Topic OId="1104nezC90m8NYAi2fjQvw=="><ap:Text PlainText="Navegadores"><ap:Font Color="ff000000" Size="12"/></ap:Text><ap:Color FillColor="ffffd966"/><ap:SubTopicShape/></ap:Topic><ap:Topic OId="1114nezC90m8NYAi2fjQvw=="><ap:Text PlainText="Msn"><ap:Font Color="ff000000" Size="12"/></ap:Text><ap:Color FillColor="ffffd966"/><ap:SubTopicShape/></ap:Topic></ap:SubTopics><ap:Text PlainText=""><ap:Font Color="ff000000" Size="12"/></ap:Text><ap:Color FillColor="fff1c232"/><ap:SubTopicShape/></ap:Topic><ap:Topic OId="9454nezC90m8NYAi2fjQvw=="><ap:Text PlainText=""><ap:Font Color="ff000000" Size="12"/></ap:Text><ap:Color FillColor="fff1c232"/><ap:SubTopicShape/></ap:Topic></ap:SubTopics><ap:Text PlainText=""><ap:Font Color="ff000000" Bold="true" Size="12"/></ap:Text><ap:Color FillColor="ffbf9000"/><ap:SubTopicShape/></ap:Topic><ap:Topic OId="2154nezC90m8NYAi2fjQvw=="><ap:SubTopics><ap:Topic OId="2554nezC90m8NYAi2fjQvw=="><ap:SubTopics><ap:Topic OId="2854nezC90m8NYAi2fjQvw=="><ap:Text PlainText=""><ap:Font Color="ff000000" Size="12"/></ap:Text><ap:Color FillColor="ffead1dc"/><ap:SubTopicShape SubTopicShape="urn:mindjet:RoundedRectangle&#10;                        "/></ap:Topic></ap:SubTopics><ap:Text PlainText=""><ap:Font Color="fffeffff" Size="12"/></ap:Text><ap:Color FillColor="ffc27ba0"/><ap:SubTopicShape SubTopicShape="urn:mindjet:RoundedRectangle&#10;                        "/></ap:Topic><ap:Topic OId="2954nezC90m8NYAi2fjQvw=="><ap:SubTopics><ap:Topic OId="3054nezC90m8NYAi2fjQvw=="><ap:Text PlainText=""><ap:Font Color="ff000000" Size="12"/></ap:Text><ap:Color FillColor="ffead1dc"/><ap:SubTopicShape SubTopicShape="urn:mindjet:RoundedRectangle&#10;                        "/></ap:Topic></ap:SubTopics><ap:Text PlainText="Salida de datos"><ap:Font Color="fffeffff" Size="12"/></ap:Text><ap:Color FillColor="ffc27ba0"/><ap:SubTopicShape SubTopicShape="urn:mindjet:RoundedRectangle&#10;                        "/></ap:Topic><ap:Topic OId="3154nezC90m8NYAi2fjQvw=="><ap:SubTopics><ap:Topic OId="3254nezC90m8NYAi2fjQvw=="><ap:Text PlainText=""><ap:Font Color="ff000000" Size="12"/></ap:Text><ap:Color FillColor="ffead1dc"/><ap:SubTopicShape SubTopicShape="urn:mindjet:RoundedRectangle&#10;                        "/></ap:Topic></ap:SubTopics><ap:Text PlainText="Almacenamiento"><ap:Font Color="fffeffff" Size="12"/></ap:Text><ap:Color FillColor="ffc27ba0"/><ap:SubTopicShape SubTopicShape="urn:mindjet:RoundedRectangle&#10;                        "/></ap:Topic></ap:SubTopics><ap:Text PlainText=""><ap:Font Color="fffeffff" Bold="true" Size="18"/></ap:Text><ap:Color FillColor="ffa64d79"/><ap:SubTopicShape SubTopicShape="urn:mindjet:RoundedRectangle&#10;                        "/></ap:Topic><ap:Topic OId="3W54nezC90m8NYAi2fjQvw=="><ap:SubTopics><ap:Topic OId="8W54nezC90m8NYAi2fjQvw=="><ap:Text PlainText="Computadora personal de escritorio o Desktop"><ap:Font Bold="true" Size="12"/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:RoundedRectangle&#10;                        "/></ap:Topic><ap:Topic OId="1054nezC90m8NYAi2fjQvw=="><ap:Text PlainText=""><ap:Font Bold="true" Size="12"/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:RoundedRectangle&#10;                        "/></ap:Topic><ap:Topic OId="1154nezC90m8NYAi2fjQvw=="><ap:Text PlainText="Laptop"><ap:Font Bold="true" Size="12"/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:RoundedRectangle&#10;                        "/></ap:Topic><ap:Topic OId="1254nezC90m8NYAi2fjQvw=="><ap:Text PlainText="Servidor"><ap:Font Bold="true" Size="12"/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:RoundedRectangle&#10;                        "/></ap:Topic><ap:Topic OId="1354nezC90m8NYAi2fjQvw=="><ap:Text PlainText="Tablet PC"><ap:Font Bold="true" Size="12"/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:RoundedRectangle&#10;                        "/></ap:Topic></ap:SubTopics><ap:Text PlainText="Tipos de computadora"><ap:Font Bold="true" Size="18"/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:RoundedRectangle&#10;                        "/></ap:Topic></ap:SubTopics><ap:Text PlainText="La computadora"><ap:Font Color="fffeffff" Bold="true" Size="24" Name="Verdana"/></ap:Text><ap:Color FillColor="ffcc0000"/><ap:SubTopicShape SubTopicShape="urn:mindjet:RoundedRectangle&#10;                        "/><ap:SubTopicsShape SubTopicsAlignment="urn:mindjet:Center" SubTopicsGrowth="urn:mindjet:Horizontal" SubTopicsGrowthDirection="urn:mindjet:AutomaticHorizontal&#10;                        " VerticalDistanceBetweenSiblings="150"/></ap:Topic></ap:OneTopic><ap:Relationships/></ap:Map>