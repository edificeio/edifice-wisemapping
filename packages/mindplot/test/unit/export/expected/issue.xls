<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<?mso-application progid="Excel.Sheet"?><Workbook xmlns:x="urn:schemas-microsoft-com:office:excel" xmlns:ss="urn:schemas-microsoft-com:office:spreadsheet" xmlns:duss="urn:schemas-microsoft-com:office:dummyspreadsheet" xmlns:o="urn:schemas-microsoft-com:office:office" xmlns="urn:schemas-microsoft-com:office:spreadsheet">
<Styles>
<Style ss:ID="s16" ss:Name="attribute_cell">
<Borders>
<Border ss:Position="Bottom" ss:LineStyle="Continuous" ss:Weight="1"/>
<Border ss:Position="Left" ss:LineStyle="Continuous" ss:Weight="1"/>
<Border ss:Position="Right" ss:LineStyle="Continuous" ss:Weight="1"/>
<Border ss:Position="Top" ss:LineStyle="Continuous" ss:Weight="1"/>
</Borders>
</Style>
<Style ss:ID="s17" ss:Name="attribute_header">
<Borders>
<Border ss:Position="Bottom" ss:LineStyle="Continuous" ss:Weight="1"/>
<Border ss:Position="Left" ss:LineStyle="Continuous" ss:Weight="1"/>
<Border ss:Position="Right" ss:LineStyle="Continuous" ss:Weight="1"/>
<Border ss:Position="Top" ss:LineStyle="Continuous" ss:Weight="1"/>
</Borders>
<Font ss:Bold="1"/>
</Style>
</Styles>
<Worksheet ss:Name="FreeMind Sheet">
<Table>
<Row>
<Cell ss:Index="1">
<Data ss:Type="String">La computadora</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="2">
<ss:Data xmlns="http://www.w3.org/TR/REC-html40" ss:Type="String">
<p>Software</p>
<p>(Programas y datos con los que funciona la computadora)</p>
</ss:Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<ss:Data xmlns="http://www.w3.org/TR/REC-html40" ss:Type="String">
<p>Software de Sistema:Permite el entendimiento</p>
<p>entre el usuario y la maquina.</p>
</ss:Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="4">
<Data ss:Type="String">Microsoft Windows</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="4">
<Data ss:Type="String">GNU/LINUX</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="4">
<Data ss:Type="String">MAC </Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<ss:Data xmlns="http://www.w3.org/TR/REC-html40" ss:Type="String">
<p>Software de Aplicación: Permite hacer hojas de</p>
<p>calculo navegar en internet, base de datos, etc.</p>
</ss:Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="4">
<Data ss:Type="String">Office</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="4">
<Data ss:Type="String">Libre Office</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="4">
<Data ss:Type="String">Navegadores</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="4">
<Data ss:Type="String">Msn</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<ss:Data xmlns="http://www.w3.org/TR/REC-html40" ss:Type="String">
<p>Software de Desarrollo</p>
</ss:Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="2">
<ss:Data xmlns="http://www.w3.org/TR/REC-html40" ss:Type="String">
<p>Hardware</p>
<p>(componentes físicos)</p>
</ss:Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<ss:Data xmlns="http://www.w3.org/TR/REC-html40" ss:Type="String">
<p>Entrada de datos</p>
</ss:Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="4">
<ss:Data xmlns="http://www.w3.org/TR/REC-html40" ss:Type="String">
<p>Ratón, Teclado, Joystick,</p>
<p>Cámara digital, Micrófono, Escáner.</p>
</ss:Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">Salida de datos</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="4">
<ss:Data xmlns="http://www.w3.org/TR/REC-html40" ss:Type="String">
<p>Monitor, Impresora, Bocinas, Plóter.</p>
</ss:Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">Almacenamiento</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="4">
<ss:Data xmlns="http://www.w3.org/TR/REC-html40" ss:Type="String">
<p>Disquete, Disco compacto, DVD,</p>
<p>BD, Disco duro, Memoria flash.</p>
</ss:Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="2">
<Data ss:Type="String">Tipos de computadora</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">Computadora personal de escritorio o Desktop</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<ss:Data xmlns="http://www.w3.org/TR/REC-html40" ss:Type="String">
<p>PDA</p>
</ss:Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">Laptop</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">Servidor</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">Tablet PC</Data>
</Cell>
</Row>
</Table>
</Worksheet>
</Workbook>
