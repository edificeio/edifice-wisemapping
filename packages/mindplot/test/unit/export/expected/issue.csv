La computadora 
,
             
,
            ,
             
,
            ,
            ,
            Microsoft Windows 
,
            ,
            ,
            GNU/LINUX 
,
            ,
            ,
            MAC  
,
            ,
             
,
            ,
            ,
            Office 
,
            ,
            ,
            Libre Office 
,
            ,
            ,
            Navegadores 
,
            ,
            ,
            Msn 
,
            ,
             
,
             
,
            ,
             
,
            ,
            ,
             
,
            ,
            Salida de datos 
,
            ,
            ,
             
,
            ,
            Almacenamiento 
,
            ,
            ,
             
,
            Tipos de computadora 
,
            ,
            Computadora personal de escritorio o Desktop 
,
            ,
             
,
            ,
            Laptop 
,
            ,
            Servidor 
,
            ,
            Tablet PC 
